From d45d9fd038ccabeec5beeb3298d54bf10917b6d5 Mon Sep 17 00:00:00 2001
From: Dom Cobley <popcornmix@gmail.com>
Date: Thu, 16 Dec 2021 14:22:37 +0000
Subject: [PATCH 1303/2201] Revert "media: bcm2835-codec: Limit video
 callbacks"

This reverts commit f814bfc5f4d3005eb266a1556be8b7b8770629bd.

The commit caused media stalls with kodi and stateful
v4l2 video decode.

John is now using a different way of limiting latency
through stateful v4l2 so this is not required.
---
 .../vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c      | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c b/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
index 4cd940211b54..fdbaa96c25a0 100644
--- a/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
+++ b/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
@@ -2536,14 +2536,6 @@ static int bcm2835_codec_create_component(struct bcm2835_codec_ctx *ctx)
 					      MMAL_PARAMETER_VIDEO_STOP_ON_PAR_COLOUR_CHANGE,
 					      &enable,
 					      sizeof(enable));
-
-		enable = (unsigned int)-5;
-		vchiq_mmal_port_parameter_set(dev->instance,
-					      &ctx->component->control,
-					      MMAL_PARAMETER_VIDEO_MAX_NUM_CALLBACKS,
-					      &enable,
-					      sizeof(enable));
-
 	} else if (dev->role == DEINTERLACE) {
 		/* Select the default deinterlace algorithm. */
 		int half_framerate = 0;
-- 
2.17.1

